plugins/inputs/mavlink/mavlink_test.go

```diff
@@ -0,0 +1,85 @@
+package mavlink
+
+import (
+	"testing"
+
+	"github.com/bluenviron/gomavlib/v3"
+	"github.com/stretchr/testify/require"
+)
+
+// Test that a serial port URL can be parsed (Linux)
+func TestParseSerialUrlLinux(t *testing.T) {
+	testConfig := Mavlink{
+		URL: "serial:///dev/ttyACM0:115200",
+	}
+
+	err := testConfig.Init()
+	require.NoError(t, err)
+	endpoint, ok := testConfig.endpointConfig[0].(gomavlib.EndpointSerial)
+	require.True(t, ok)
+	require.Equal(t, "/dev/ttyACM0", endpoint.Device)
+	require.Equal(t, 115200, endpoint.Baud)
```

Comment:
Please be brief! It also helps readability to call the plugin instance `plugin`... ;-)

```suggestion
	plugin := &Mavlink{
		URL: "serial:///dev/ttyACM0:115200",
	}
	require.NoError(t, plugin.Init())

	endpoint, ok := plugin.endpointConfig.(gomavlib.EndpointSerial)
	require.True(t, ok)
	require.Equal(t, "/dev/ttyACM0", endpoint.Device)
	require.Equal(t, 115200, endpoint.Baud)
```

Same for the tests below.

Reply:
Updated!